demo/src/example/example-page.js

```diff
@@ -8,67 +8,215 @@ import router from '../core/router';
+    this.initDrmSettings();
+  }
+
 
-    // Add event listener for the 'click' event on the '#examples' element.
-    document.querySelector('#examples').addEventListener('click', (event) => {
+  /**
+   * Initializes the examples listeners. Whenever an example button is clicked,
+   * a modal with a player is opened and the corresponding example's content is loaded.
+   */
+  initExamples() {
+    // Listen for example's click
+    document.querySelector('#examples').addEventListener('click', event => {
       const exampleEl = event.target.closest('button');
 
-      // Check if the clicked element is a button and has the 'category' dataset attribute.
-      if (!exampleEl || !('category' in exampleEl.parentNode.dataset)) {
+      // Check if the clicked element is a button and has the 'section' dataset attribute.
+      if (!exampleEl || !('section' in exampleEl.parentNode.dataset)) {
         return;
       }
 
-      this.onExampleClicked(exampleEl);
+      const parent = exampleEl.parentNode;
+      const section = parent.dataset.section;
+      const exampleIdx = Array.from(parent.children)
+        .filter(child => child.tagName.toLowerCase() === 'button')
+        .indexOf(exampleEl);
+
+      openPlayerModal(Examples[section][exampleIdx]);
     });
+  }
+
+  /**
+   * Initializes the load bar element.
+   */
+  initLoadBar() {
+    this.#playContentBtnEl.addEventListener('click', () => this.playCustomUrl());
+
+    this.#loadBarEl.addEventListener('keyup', event => {
+      const src = this.customSrc;
 
-    // Add event listener for the 'keyup' event on the '#load-bar' element.
-    document.querySelector('#load-bar').addEventListener('keyup', (event) => {
+      this.#playContentBtnEl.disabled = !src;
       // Check if the 'Enter' key is pressed.
-      if (event.key === 'Enter') {
-        this.onSourceSubmitted(event.target.value);
+      if (event.key === 'Enter' && src) {
+        this.playCustomUrl();
       }
     });
   }
 
   /**
-   * Handles the click event on an example button. A modal with a player is opened
-   * and the corresponding example's content is loaded.
+   * Get the currently configured key systems in the DRM settings of the load bar.
    *
-   * @param {Element} exampleEl - The button that was clicked.
+   * @returns {{}|undefined} the key systems configured by the user.
    */
-  onExampleClicked(exampleEl) {
-    const parent = exampleEl.parentNode;
-    const category = parent.dataset.category;
-    const exampleIdx = Array.from(parent.children)
-      .filter(child => child.tagName.toLowerCase() === 'button')
-      .indexOf(exampleEl);
-
-    openPlayerModal(Examples[category][exampleIdx]);
+  get keySystems() {
+    const vendor = this.#drmVendorEl.value;
+
+    if (!vendor) {
+      return undefined;
+    }
+
+    const keySystems = {};
+    const certificateUrl = this.#certificateUrlEl.value;
+    const licenseUrl = this.#licenseUrlEl.value;
+
+    if (Drm.vendors.FAIRPLAY === vendor) {
```

Comment:
pls refer to my comment above.

Reply:
Refer to my answer above to your comment above :P